composer.json

```diff
@@ -27,16 +27,20 @@
         "classmap": ["src/Omnipay/Omnipay.php"]
     },
     "require": {
-        "php": ">=5.3.2",
-        "guzzle/guzzle": "~3.9",
-        "symfony/http-foundation": "~2.1"
+        "php": ">=5.5.9",
+        "symfony/http-foundation": "^2.1|^3.0",
```

Comment:
Fixed in https://github.com/thephpleague/omnipay-common/commit/f785808a4d9984d6d3717c13f2636977a6b58ef2